pkg/source/clients/ossprotocol/oss_source_client.go

```diff
@@ -320,20 +321,31 @@ func (osc *ossSourceClient) getClient(header source.Header) (*oss.Client, error)
 	if pkgstrings.IsBlank(accessKeySecret) {
 		return nil, errors.New("accessKeySecret is empty")
 	}
-	clientKey := buildClientKey(endpoint, accessKeyID, accessKeySecret)
+	securityToken := header.Get(securityToken)
+	clientKey := buildClientKey(endpoint, accessKeyID, accessKeySecret, securityToken)
 	if client, ok := osc.clientMap.Load(clientKey); ok {
 		return client.(*oss.Client), nil
 	}
-	client, err := oss.New(endpoint, accessKeyID, accessKeySecret)
+	var client *oss.Client
+	var err error
+	if pkgstrings.IsBlank(securityToken) {
+		client, err = oss.New(endpoint, accessKeyID, accessKeySecret)
+	} else {
+		client, err = oss.New(endpoint, accessKeyID, accessKeySecret, oss.SecurityToken(securityToken))
```

Comment:
Just return the client if err == nil.
The securityToken is temporary, so the client is same.